src/core/public/public.api.md

```diff
@@ -192,6 +192,10 @@ export interface HttpServiceBase {
     getLoadingCount$(): Observable<number>;
     // (undocumented)
     head: HttpHandler;
+    // Warning: (ae-forgotten-export) The symbol "HttpInterceptor" needs to be exported by the entry point index.d.ts

```

Comment:
nit: should probably export HttpInterceptor from core/public

Reply:
Ah good catch!